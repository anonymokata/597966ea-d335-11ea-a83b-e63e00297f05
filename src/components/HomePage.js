import React from 'react';

import TimeForm from './TimeForm';


const HomePage = () => (
  <div>
    <h1>Kata</h1>
    <TimeForm />
  </div>
);

export default HomePage;