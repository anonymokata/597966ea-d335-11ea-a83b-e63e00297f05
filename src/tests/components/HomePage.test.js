import React from 'react';
import { shallow } from 'enzyme';

import HomePage from '../../components/HomePage';

test('Should render Homepage correctly', () => {
  const wrapper = shallow(<HomePage />);
  expect(wrapper).toMatchSnapshot();
});