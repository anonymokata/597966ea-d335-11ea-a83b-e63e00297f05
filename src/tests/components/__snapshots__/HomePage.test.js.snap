// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Homepage correctly 1`] = `
<div>
  <h1>
    Kata
  </h1>
  <TimeForm />
</div>
`;
